spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 $LOAD_PATH.unshift('./lib/')
 
 ProcessStatusStub = Struct.new(:exitstatus)
-$STATUS_ZERO = ProcessStatusStub.new(0)
-$STATUS_ONE = ProcessStatusStub.new(1)
+STATUS_OK = ProcessStatusStub.new(0)
+STATUS_ERROR = ProcessStatusStub.new(1)
 
+ENV['PATH'] = "./spec/adb_mock/:" + ENV['PATH']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.